tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_browser_play_video.py

```diff
@@ -0,0 +1,70 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from marionette_driver import Wait
+
+from gaiatest.apps.search.app import Search
+from gaiatest.apps.search.regions.html5_player import HTML5Player
+from gaiatest.apps.homescreen.regions.permission_dialog import PermissionDialog
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+
+class TestVideo(GaiaImageCompareTestCase):
+
+    def setUp(self):
+        GaiaImageCompareTestCase.setUp(self)
+        self.connect_to_local_area_network()
+        self.video_URL = self.marionette.absolute_url('VID_0001.ogg')
+
+    def test_play_video(self):
+        """Confirm video playback
+
+        https://moztrap.mozilla.org/manage/case/6073/
```

Comment:
The steps below don't reflect the entire test case here. I'd add a comment to warn any future readers about that.